src-java/kilda-persistence-api/src/main/java/org/openkilda/persistence/tx/TransactionManager.java

```diff
@@ -15,12 +15,44 @@
 
 package org.openkilda.persistence.tx;
 
+import org.openkilda.persistence.context.PersistenceContextRequired;
+import org.openkilda.persistence.exceptions.PersistenceException;
+import org.openkilda.persistence.exceptions.RecoverablePersistenceException;
+
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import net.jodah.failsafe.Failsafe;
+import net.jodah.failsafe.FailsafeException;
 import net.jodah.failsafe.RetryPolicy;
 
+import java.io.Serializable;
+import java.time.temporal.ChronoUnit;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.Set;
+import java.util.concurrent.Callable;
+
 /**
  * Manager of transaction boundaries.
  */
-public interface TransactionManager {
+@Slf4j
+public class TransactionManager implements Serializable {
+    private static final ThreadLocal<ActiveAdapters> globals = new ThreadLocal<>();
```

Comment:
`globals` should be uppercase.

Reply:
I don't agree to treat it as a constant (also it is private, also out checkstyle do not argue against it), but... let it be all uppercase.